### PR TITLE
chore: prepared push-notification package for release

### PR DESCRIPTION
**Pull request by tolerants**
- Created 2 new workflows for release: minor and patch.
- Prepared push-notification package for release.

closes https://spryker.atlassian.net/browse/HRZ-2677

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "sf:e2e:headless:ci:affected": "npx nx print-affected --select=projects | grep -E -v 'storefront' && echo 'nothing changed, skip' || npm run sf:e2e:headless:ci",
     "release": "run-s release:*",
     "release-labs": "cd libs && npm run release-labs",
+    "release-patch": "cd libs && npm run release-patch",
     "release:libs": "cd libs && npm run release",
```
Comment (AndreyYevtukhov) [inline package.json:25]: `release:libs` -> `release-libs` with a dash to be consistent? 
or it was done on purpose?

Comment (AndreyYevtukhov) [reply package.json:25]: or maybe to change `release-labs` to `release:labs`?

File: .github/workflows/release-minor.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Build packages
+        run: npx nx run-many --target=build --all --parallel=2 --exclude storefront,fulfillment
+      - name: Fix js extension
+        run: node tools/post-build/add-js-extension.js
+      - name: Release patch
+        if: ${{ startsWith(env.RELEASE_BRANCH, 'refs/heads/release/') }}
+        run: npm run minor
+      - name: Release Error Log
+        if: failure()
+        run: cat libs/lerna-debug.log
```
Comment (AndreyYevtukhov) [inline .github/workflows/release-minor.yml:28]: Is it possible that sensitive info (aka keys) will be logged and visible to the public?

**Review by AndreyYevtukhov — COMMENTED** [review]
Left some small comments

File: package.json (hunk 1 above)
Comment (tolerants) [reply package.json:25]: It was done on purpose. `release:libs` - is kind of sub-command of `release`, so with `run-s release:*` command we are running all the commands starts with `release:`, but `release-labs` is a different script.

File: .github/workflows/release-minor.yml (hunk 2 above)
Comment (tolerants) [reply .github/workflows/release-minor.yml:28]: Only for someone who has an access to ci.

**Review by AndreyYevtukhov — DISMISSED** [review]

**Review by AndreyYevtukhov — APPROVED** [review]

**Pull request merged**